packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -168,6 +167,7 @@ async function updateWorkerBundledCode(
 
   patchedCode = patchedCode
     // workers do not support dynamic require nor require.resolve
+    // TODO: implement for cf
```

Comment:
Maybe add a note that this can be done in `@opennextjs/aws`, not necessarily here